actor/v7action/service_instance_details.go

```diff
@@ -32,7 +31,7 @@ type SharedStatus struct {
 	UsageSummary              []UsageSummaryWithSpaceAndOrg
 }
 
-type ServiceInstanceParameters types.JSONObject
```

Comment:
The comment for this type says:
```go
// JSONObject represents a JSON object. When not initialized it serializes to `{}`,
// whereas a `map[string]interface{}` serializes to `null`.
```
So this is a change of behaviour. Is that intended?

Reply:
I think in this all the usages we initialise it. So it shouldn't be an issue. 
Also, the serialization was happening via json.MarshalIndent, which was the one causing the issue.